src/odemis/util/conversion.py

```diff
@@ -31,49 +31,38 @@
 import re
 import yaml
 
-# Inspired by code from:
-# http://codingmess.blogspot.nl/2009/05/conversion-of-wavelength-in-nanometers.html
-# based on:
-# http://www.physics.sfasu.edu/astro/colour/spectra.html
-def wave2rgb(wavelength):
+
+def wavelength2rgb(wavelength):
     """
-    Convert a wavelength into a (r,g,b) value
-    wavelength (0<float): wavelength in m
-    return (3-tupe int in 0..255): RGB value
+    Convert a wavelength into a (r, g, b) value
+    wavelength (0 < float): wavelength in m
+    return (3-tuple int in 0..255): RGB value
+
+    Notes:
+    Inspired by code from:
+    http://codingmess.blogspot.nl/2009/05/conversion-of-wavelength-in-nanometers.html
+    based on:
+    http://www.physics.sfasu.edu/astro/color/spectra.html
```

Comment:
if this link doesn't work maybe we should hunt down a working one or write a note that it doesn't work.

Reply:
I already did, and this is the working one